schedule/yast/libyui_hdd.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name:          create_hdd_gnome_libyui
+description:    >
+    Boot into gnome HDD and republish image for tests that use libyui.
+vars:
+   SCC_ADDONS: sdk
+   INSTALL_PACKAGES: libyui-rest-api
+schedule:
+   - boot/boot_to_desktop
+   - console/scc_cleanup_reregister
+   - console/install_packages_simple
+   - console/firewalld_add_port
+   - shutdown/shutdown
+test_data:
+   install_packages: libyui-rest-api
```

Comment:
it is more clear if we use a list here as we have been doing for other cases (so it is obvious and you don't need to specify the special format of package separated by spaces in the header):
```
install_packages:
  - libyui-rest-api
```

Reply:
Yes , it would be a bit more clear. I would still have to clarify this in the module comment. Plus, I would have to join the key values in one, or create a loop. The question is what is better? Cleaner code or test data?

Reply:
I don't think you need to clarify anything in the module, because we would be reading an array, we use `join` (one line) in many places to provide input in the way that command need it. The code is still clean with the join and you don't need to provide extra information about how to format the test data in the test module and the test data is more clear for sure. wdyt?

Reply:
For me, both ways are ok. I went with the first one, as more simple working solution. I can change it.

Reply:
if simple it is, my only concern is the pattern that we use, for this case should be ok either way.
IMO if we use list in test data, we should use proper list in yaml, otherwise we would end up with a long line there if many packages, another disadvantage I see is that I need to read the header of the module to know that the special format is comma-separated and not space-separated or other format.
You can leave it as it is, my only advice is to not look for the most simple solution but the most simple that scale well (in other words, that someone else can borrow for other problem and still be fine)